impl_core/src/main/java/io/opencensus/implcore/trace/propagation/BinaryFormatImpl.java

```diff
@@ -77,7 +77,7 @@
       4 * ID_SIZE + TraceId.SIZE + SpanId.SIZE + TraceOptions.SIZE;
 
   @Override
-  public byte[] toBinaryValue(SpanContext spanContext) {
+  public byte[] toByteArray(SpanContext spanContext) {
```

Comment:
Shouldn't BinaryFormatImpl continue to implement to/fromBinaryValue instead, for backwards compatibility with the API?

Reply:
For me backwards compatibility means a code instrumented with API 0.6.0 can run with API 0.7.0 without a problem. 

Based on how dependency management work, (you can read more https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html) the final binary controls what API/implementation is used so we don't need to worry about Impl 0.7.0 working with API 0.6.0 (that's also forward compatibility I feel and we do not guarantee that).

Reply:
I thought that the implementation also needed to be compatible with older versions of the API for some period of time, in case users don't exclude the API dependency.  I'll ask you about it later, though.